### ISSUE TITLE
Importing highlights into Obsidian in wrong order

### ISSUE THREAD

**Issue by shisuuu** [aidenlx/obsidian-zotero#72]
Hi!
First I want to say that I love this plugin since it complements my workflow!

I have a problem when importing highlights using "Open quick switcher for literature notes"  though.
Whenever I'm trying to import them into a note, the order is totally messed up.
For example I have a highlight on page 44 shown as the first zotero annotation, followed by random page numbers.

My question is whether that might be a problem of the plugin or is it on my side?

Thanks for creating this plugin :)

**Comment by lkhphuc** [comment]
For now you can sort it yourself in the template. 
My template for reference `zt-note.eta`:
```js
##### <% it.creators.sort((a, b) => (a.orderIndex - b.orderIndex)).forEach(function(creator){ %><%- =  creator.fullname%>, <% }) %>(<%-= it.date %>). *<%-= it.title %>*

> [!abstract]
> <%= it.abstractNote %>

- #type/<%= it.itemType%> | [Local Library](<%= it.backlink %>) | [URL](<%= it.url%>)

<%~ include("annots", it.annotations.sort((a, b) => ((a.sortIndex[0] - b.sortIndex[0]) || (a.sortIndex[1] - b.sortIndex[1]) || (a.sortIndex[2] - b.sortIndex[2])))) %>

```

**Comment by aidenlx** [comment]
Fixed in latest 1.0.0 plugin
